### ISSUE TITLE
Confusing examples

### ISSUE THREAD

**Issue by wrathematics** [RBigData/pbdIO#4]
I don't see much value in having "real" examples in the demo/ tree.  This feels much better suited for the vignette, or possibly pbdDEMO.  The package is for I/O, so adding a layer of analysis on top doesn't make much sense to me, especially since you copied code from pbdDEMO; prefer not to go down that road.

**Comment by go-ski** [comment]
Agreed. But let's hold off on that until I finish the matrix.r example in the next few days. Then move both.

**Comment by wrathematics** [comment]
Consider moving these to the vignette now that the skeleton is there.
